src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpContentReadStream.cs

```diff
@@ -28,6 +28,8 @@ public HttpContentReadStream(HttpConnection connection) : base(connection)
 
             public virtual bool NeedsDrain => false;
 
+            internal bool IsDisposed => _disposed == 1;
```

Comment:
protected instead of internal? The class itself is already internal.

Reply:
Will change.